be/src/column/binary_column.h

```diff
@@ -335,6 +343,14 @@ class BinaryColumnBase final : public ColumnFactory<Column, BinaryColumnBase<T>>
 private:
     void _build_slices() const;
 
+    void _check_overvlow() const {
+        if constexpr (std::is_same_v<T, uint32_t>) {
+            if (UNLIKELY(_bytes.size() > std::numeric_limits<uint32_t>::max())) {
```

Comment:
we shouldn't throw exception any where. some scope was not in catched scope

Reply:
updated